frontend/src/scenes/groups/GroupsIntroduction.tsx

```diff
@@ -36,6 +36,7 @@ export function GroupsIntroduction({ access }: Props): JSX.Element {
             featureKey="group-analytics"
             caption={subtext}
             docsLink="https://posthog.com/docs/user-guides/group-analytics"
+            hideUpgradeButton={access === GroupsAccessStatus.HasAccess}
```

Comment:
Note to @paolodamico: This was broken by https://github.com/PostHog/posthog/commit/df49e7ae77adbe34071b954ec3b7f519fa938956

The other thing that commit broke was primary/secondary button logic - when groups access exists, primary button should be the docs button according to the original design.

Worth revisiting this design a bit?

Reply:
Yup I'll take a look